rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/nodes.stats/90_fs_watermark_stats.yml

```diff
@@ -1,10 +1,5 @@
 ---
 "Allocation stats":
-  - requires:
-      cluster_features: ["stats.include_disk_thresholds"]
-      reason: "fs watermark stats was added in 8.15.0"
-      test_runner_features: [arbitrary_key]
```

Comment:
We'll likely still need the `test_runner_features`, right?

Reply:
Oh yes, and that's why reviews are needed...